cms/templates/import.html

```diff
@@ -48,11 +72,23 @@ <h3 class="title">
 
           <input type="file" name="course-data" class="file-input" />
 
-          <input type="submit" value="${_('Replace my course with the one above')}" class="submit-button" />
+          <input type="submit"
+          %if library:
+              value="${_('Replace my library with the one above')}"
```

Comment:
I'd like to suggest revised wording: "Replace my library with the selected file"
@marcotuts your thoughts?